src/configuration/services/network-storage.md

```diff
@@ -0,0 +1,170 @@
+# Network Storage
+
+Platform.sh supports internal "storage as a service" to provide a file store that can be shared between different application containers.
+
+The network storage service enables a new kind of `mount` that refers to a shared service rather than to a local directory.  Any application can use both `local` or `service` mounts, or either/or, or neither.
```

Comment:
>  Any application can use both local or service mounts,or either/or, or neither.

I would suggest to rephrase to `Any application can use both local and/or service mounts, or neither.`. The original `or either/or, or neither` looks very confusing...